src/components/ManageTask.vue

```diff
@@ -72,37 +128,49 @@ let taskName = ref("")
 let taskLink = ref("")
 let taskPoint = ref(null)
 
+//is alert show
 let isShow = ref(false)
 
+let alertMessage = ref("")
+let alertBtnMessage = ref("OK")
+
 function addTask() {
   //validation
-  if(taskName.value.trim() === "") {
+  if (taskName.value.trim() === "") {
     //TODO alert
     isShow.value = true
+    alertMessage.value = TASK_NAME_IS_BLANK
     // alert(TASK_NAME_IS_BLANK)
   } else {
     const task = {
       id: taskArray1.value.length + taskArray2.value.length + 1,
-      name: taskName,
+      name: taskName.value,
       link: taskLink,
-      points: taskPoint
+      points: taskPoint,
+      isToggleOpen: false
     }
     taskArray1.value.push(task)
   }
 }
 
 function deleteTask(id) {
-  if(taskArray1.value.filter(t => t.id == id).length > 0) {
+  if (taskArray1.value.filter(t => t.id == id).length > 0) {
     taskArray1.value = taskArray1.value.filter(t => t.id != id)
   } else {
     taskArray2.value = taskArray2.value.filter(t => t.id != id)
   }
 }
 
 function submit() {
-  // TODO check taskArray2 is not empty
-  localStorage.removeItem(TASKS_KEY)
-  localStorage.setItem(TASKS_KEY, JSON.stringify(taskArray2.value))
+  if (taskArray2.value.length === 0) {
+    //TODO alert
+    isShow.value = true
+    alertMessage.value = ORDER_IS_EMPTY
```

Comment:
1. remove the unused comment
2. alertMessage.value = ORDER_IS_EMPTY should before isShow.value = true